tools/make_hgrid/create_gnomonic_cubic_grid.c

```diff
@@ -94,51 +139,103 @@ void create_gnomonic_cubic_grid( char* grid_type, int *nlon, int *nlat, double *
   nj  = ni;
   nip = ni+1;
   njp = nip;
-  ni_nest = 0;
-  nj_nest = 0;  
+
+  for (nn=0; nn < MAX_NESTS; nn++) {
+    ni_nest[nn] = 0;
+    nj_nest[nn] = 0;  
+  }
+
   ntiles2=ntiles;
   global_nest=0;
-  if(nest_grid && parent_tile== 0)
+
+  if (verbose) fprintf(stderr, "[INFO] B\n");
```

Comment:
This `fprintf` appears to be a leftover debug statement.  If you want a info statement here, it should be than just "B", or just remove this line.

Reply:
I think I've removed all these short print statements in the latest commit. I also took out the old commented-out code that you mentioned in a comment below.